lib/hterm.js

```diff
@@ -188,7 +189,7 @@ lib.colors.hexToRGB = function (arg) {
   }
 
   if (arg instanceof Array) {
-    for (var i = 0; i < arg.length; i++) {
+    for (let i = 0; i < arg.length; i++) {
```

Comment:
Here it would be better to use a `for..of` loop.

No rule for it [yet](https://github.com/sindresorhus/eslint-plugin-unicorn/issues/49), but consider me a linter :p